### PR TITLE
Browser cache invalidation improvements

### PR DESCRIPTION
**Pull request by pekura**

### CODE REVIEW

**Comment by parostatkiem** [comment]
Did you forget the PR?

**Comment by pekura** [comment]
This is a quick solution for the 0.7 release. The combination of etags and cache max-age makes it a lot better and performance is acceptable. Maximum performance would be achieved by a version number or content hash based id as part of the name of each resource but this requires more time to discuss and implement - we might improve for the 0.8 release.

File: lambda/nginx/nginx.conf [hunk 1]
```diff
@@ -39,6 +39,8 @@ http {
             root   /usr/share/nginx/html;
             index  index.html index.htm;
             try_files $uri$args $uri$args/ /index.html;
+
```
Comment (mjasinski5) [inline lambda/nginx/nginx.conf]: In other views there's also 
`add_header Access-Control-Allow-Origin *;` to enable cors. Is this intended to omit this line?

File: brokers/nginx.conf [hunk 2]
```diff
@@ -26,11 +26,13 @@ http {
...
     root /var/public;
     port_in_redirect off;
-    add_header Access-Control-Allow-Origin *;
 
     location / {
 
         try_files $uri$args $uri$args/ $uri $uri/ /;
+
+        add_header 'Cache-Control' 'public, max-age=300';
+        add_header Access-Control-Allow-Origin *;
```
Comment (mjasinski5) [inline brokers/nginx.conf]: Is that intended to disable cors on /status endpoint?

File: lambda/nginx/nginx.conf (hunk 1 above)
Comment (pekura) [reply lambda/nginx/nginx.conf]: This header is not necessary for (microfrontend) views, I did not add or remove it here and tried not to touch it in the other views as well.

File: brokers/nginx.conf (hunk 2 above)
Comment (pekura) [reply brokers/nginx.conf]: No, that was not my intention, I did not want to change the behaviour for this header. I re-added it for /status endpoint, thanks for pointing it out!

**Pull request merged**